cw/cw305/capture.py

```diff
@@ -227,45 +269,20 @@ def capture_aes_random_batch(ot, ktp, capture_cfg, scope_type):
             # Check that the ADC didn't saturate when recording this batch.
             check_range(waves, ot.scope.adc.bits_per_sample)
 
-            # Generate plaintexts and ciphertexts to compare with the batch encryption results.
+            # Generate plaintexts and the last ciphertext in the batch
             plaintexts = [ktp.next()[1] for _ in range(scope.num_segments_actual)]
-            ciphertexts = [
-                bytearray(c)
-                for c in scared.aes.base.encrypt(
-                    np.asarray(plaintexts), np.asarray(key)
-                )
-            ]
-            # Check the last ciphertext of this batch to make sure we are in sync.
-            actual_last_ciphertext = ot.target.simpleserial_read("r", 16, ack=False)
-            expected_last_ciphertext = ciphertexts[-1]
-            assert actual_last_ciphertext == expected_last_ciphertext, (
-                f"Incorrect encryption result!\n"
-                f"actual: {actual_last_ciphertext}\n"
-                f"expected: {expected_last_ciphertext}"
+            expected_last_ciphertext = np.asarray(scared.aes.base.encrypt(
+                np.asarray(plaintexts[-1]), np.asarray(key))
```

Comment:
We actually need this for some attack scripts (e.g. `ceca.py` can be applied on the outputs for which the ciphertexts of all blocks are needed) and if I remember correctly as long as the key remeins the same, the performance impact isn't huge.

So, the PR should be changed to disable the ciphertext generation on the host only for `aes_fvsr_batch()` but still allow to enable that it turns out we need it.